packages/block/src/block.ts

```diff
@@ -137,6 +147,65 @@ export class Block {
     return new Block(header, transactions, uncleHeaders, opts)
   }
 
+  /**
+   * Creates a new block object from Ethereum JSON RPC.
+   *
+   * @param blockParams - Ethereum JSON RPC of block (eth_getBlockByNumber)
+   * @param uncles - Optional list of Ethereum JSON RPC of uncles (eth_getUncleByBlockHashAndIndex)
+   * @param options - An object describing the blockchain
+   */
+  public static fromRPC(blockData: any, uncles?: any[], opts?: BlockOptions) {
```

Comment:
blockParams or blockData? And why is this `any` typed?

Reply:
Good catch.  I meant to update this using `JsonRpcBlock`.  Now updated